### DISCUSSION TITLE
how to set the rabbitmq ports to my aspires' container like docker desktop

### DISCUSSION THREAD

**Discussion by skyDuanXianBing** [microsoft/aspire#3742] [Q&A] [answered]
like this 
![image](https://github.com/dotnet/aspire/assets/164180639/ea369b4d-096c-4650-90dd-aa1c0280e829)

**Comment by davidfowl** [comment]
What's the question?

**Reply by skyDuanXianBing** [reply]
How do I set up my ports in aspire like I do in docker desktop

**Reply by davidfowl** [reply]
oh this doc explains the networking https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/networking-overview

**Reply by davidfowl** [reply, marked as answer]
Specifically the rabbit ma resource has an AddRabbitMQ method overload with a port that you can specify.

**Reply by skyDuanXianBing** [reply]
thank you , i get try
